node_modules/oae-admin/tenants/tenants.html

```diff
@@ -121,8 +42,99 @@
             </div>
         </div>
     {/if}
+    {var isOnGlobalAdminServer = context.isGlobalAdminServer || context.isTenantOnGlobalAdminServer}
+    {if context.isGlobalAdminServer}
+        <div id="tenants-search">
+            <form action="." role="search" aria-label="__MSG__SEARCH__">
+                <label for="tenants-search-query" class="sr-only">__MSG__NARROW_BY_KEYWORD__</label>
+                <input type="text" id="tenants-search-query" class="form-control" title="__MSG__NARROW_BY_KEYWORD__" autofocus placeholder="__MSG__NARROW_BY_KEYWORD__" />
+            </form>
+        </div>
+    {/if}
+    <div id="tenants-table">
```

Comment:
It doesn't look like there's anything dynamic in this, so it can probably be moved outside of the template